_templates/_page_openshift.html.erb

```diff
@@ -513,6 +528,7 @@
     'openshift-builds' : ['docs_builds', version],
     'openshift-gitops' : ['docs_gitops', version],
     'openshift-lightspeed' : ['docs_lightspeed', version]
```

Comment:
```suggestion
    'openshift-lightspeed' : ['docs_lightspeed', version],
```

Reply:
Thank you, @aireilly . Just pushed the change.